### DISCUSSION TITLE
What is the difference between using fold=0,1,2,3,4 and fold = 5?

### DISCUSSION THREAD

**Discussion by erockman** [MIC-DKFZ/nnUNet#1029] [Q&A]
Hi Fabian,

Many thanks for providing the nnU-net for us all to use.

What does nnUNet to do when I using fold = 5 in training?
What is the difference between using fold=0,1,2,3,4 and fold = 5?

**Comment by FabianIsensee** [comment]
It should tell you what it does in the output of the training script: a random 80:20 split of the training cases
